apps/researcher/src/app/[locale]/objects/[id]/gallery.tsx

```diff
@@ -18,9 +18,10 @@ interface Props {
     src: string;
     alt: string;
   }[];
+  resetText: string;
```

Comment:
What is a 'reset text', or why and when do you need to reset a text?

Reply:
This is the text in the zoom reset button. I have renamed it to `resetZoomButton` and imported it directly into the gallery component. This is possible since [this pull request](https://github.com/colonial-heritage/colonial-collections/pull/172).